geonode_mapstore_client/client/js/components/DetailsPanel/DetailsInfo.jsx

```diff
@@ -167,20 +168,19 @@ function DetailsInfo({
                 items: isDefaultTabType(tab.type) ? parseTabItems(tab?.items) : tab?.items,
                 Component: tabTypes[tab.type] || tabTypes.tab
             }))
-        .filter(tab => tab?.items?.length > 0);
+        .filter(tab => !isEmpty(tab?.items));
     const selectedTabId = filteredTabs?.[0]?.id;
+    const [select, onSelect] = useState(selectedTabId);
     return (
         <Tabs
             className="gn-details-info tabs-underline"
             selectedTabId={selectedTabId}
+            activeKey={select}
```

Comment:
We don't need an additional activeKey because there is already selectedTabId that has the same role